src/Plugin/Block/FooterBlock.php

```diff
@@ -81,9 +93,59 @@ public function build() {
 
     NestedArray::setValue($build, ['#corporate_footer', 'bottom_links'], $config->get('bottom_links'));
 
+    $general_links = $this->retriveCustomFooterLinks('footer_general_link', $cache);
+    $social_links = $this->retriveCustomFooterLinks('footer_social_link', $cache);
+    $site_info_config = $this->configFactory->get('system.site');
+    $cache->addCacheableDependency($site_info_config);
+    $site_identity = $site_info_config->get('name');
+
+    if (!empty($site_identity) && !empty($social_links) && !empty($general_links)) {
```

Comment:
@sergepavle Confirmed with PO. We show the footer if **EITHER** of the link types are filled in. And the site identifier can also be shown in this case (site name). There will be a followup to make it it more configurable.